build.sbt

```diff
@@ -306,6 +323,22 @@ def nativeImageDockerSettings: Seq[Setting[_]] = dockerSettings ++ Seq(
   }
 )
 
+def assemblySettings(jarName: String) =
+  Seq(
+    mainClass in assembly := (mainClass in Compile).value,
+    assemblyJarName in assembly := jarName,
+    test in assembly := {},
+    // logLevel in assembly := Level.Debug,
+    assemblyMergeStrategy in assembly := {
+      /*ADD CUSTOMIZATIONS HERE*/
+      case PathList("META-INF", "io.netty.versions.properties") => MergeStrategy.last
+      case PathList(ps @ _*) if ps.last endsWith ".proto" => MergeStrategy.last
```

Comment:
When we create "fat jars" it doesn't make sense to add the protos, as we don't expect anything to load them from the jar.